esp/src/src-react/components/forms/RemoteCopy.tsx

```diff
@@ -89,6 +93,23 @@ export const RemoteCopy: React.FunctionComponent<RemoteCopyProps> = ({
         }
     );
 
+    React.useEffect(() => {
+        WsTopology.TpGroupQuery({
+            request: {}
+        }).then(response => {
+            const groups = response.TpGroupQueryResponse.TpGroups?.TpGroup ?? [];
+            for (const index in groups) {
+                if (groups[index].Name === selectedDestGroup && groups[index].ReplicateOutputs === true) {
```

Comment:
Would be cleaner to have the `groups[index].Name === selectedDestGroup` test in a single outer if block?